package.json

```
@@ -33,7 +33,7 @@
     "changelog": "github_changelog_generator && git add CHANGELOG.md && git commit -am \"Updating changelog\"",
     "lint": "semistandard --fix",
     "mocha": "mocha --opts mocha.opts",
-    "coverage": "istanbul cover node_modules/mocha/bin/_mocha -- --opts mocha.opts",
+    "coverage": "nyc report node_modules/mocha/bin/_mocha -- --opts mocha.opts",
```

nyc report only runs previously defined reports and therefor fails (does not create a .nyc_output)

```suggestion
    "coverage": "nyc --check-coverage npm run mocha",
```